Intersect.Client/Interface/Game/Inventory/InventoryWindow.cs

```diff
@@ -283,15 +283,15 @@ public bool IsVisible()
             return !mInventoryWindow.IsHidden;
         }
 
-        public bool Hide()
+        public void Hide()
         {
             if (!Globals.CanCloseInventory)
             {
-                return false;
```

Comment:
Nothing uses the return value?

Reply:
> Nothing uses the return value?

![imagen](https://user-images.githubusercontent.com/17498701/201738023-590e792d-671e-476a-a9dc-a5a02f511ece.png)


Hide() was acting as a boolean but nothing used to require / currently requires it's previously returned values, so i turned the method into a void one since it doesn't really requires to return a value where it's used